README.md

```diff
@@ -14,8 +14,29 @@ named `SparkModel` that extends the Pydantic's `BaseModel`.
 
 ## Features
 
-- Conversion from Pydantic model to PySpark schema
+- Conversion from Pydantic model to PySpark schema (`StructType` or JSON)
 - Type coercion
+- PySpark as an optional dependency
+
+## Installation
+
+Without PySpark:
+
+```shell
+pip install sparkdantic
+```
+
+> Note: only **JSON** schema generation features are available without PySpark installed. If you attempt to use PySpark-dependent schema generation features, SparkDantic will check that a supported version of Pyspark is installed.
+
+With PySpark:
+
+```shell
+pip install sparkdantic[pyspark]
```

Comment:
This should be `pip install "sparkdantic[pyspark]"`

Reply:
Nice spot! Fixed.